lib/smack/SmackInstGenerator.cpp

```diff
@@ -623,6 +623,16 @@ void SmackInstGenerator::visitCallInst(llvm::CallInst& ci) {
     Value* val = ci.getArgOperand(0);
     emit(Stmt::assign(rep->expr(&ci), rep->expr(val)));
 
+  } else if (SmackOptions::BitPrecise && SmackOptions::FloatEnabled &&
```

Comment:
What code will get generated (if any) if BitPrecise and FloatEnabled are not turned on?
Should we not generate some kind of uninterpreted functions in that cases?

Reply:
Calls to procedures without bodies will be generated if these flags are not turned on. I think our implementations only make sense if these flags are enabled because if they are not, `i16` is unbounded, which means functions like `i16->half` are undefined.